src/main/java/life/qbic/expdesign/io/MHCLigandDesignReader.java

```diff
@@ -35,6 +35,7 @@ public class MHCLigandDesignReader implements IExperimentalDesignReader {
   private List<String> mandatoryFilled;
   private List<String> optionalCols;
   private Map<SampleType, Map<String, String>> headersToTypeCodePerSampletype;
+  private Map<String, Set<String>> parsedCategoriesToValues;
```

Comment:
Why not
```suggestion
  private Map<Category, Set<Value>> parsedCategoriesToValues;
```
What values are this?

Reply:
If you use String, Integer, Boolean,... in a map, it is not really clear what the map maps.
For readability it could be worth thinking about what actually is achieved with the map. (not clear to me)